### PR TITLE
Change the url of the API to add /api.

### PR DESCRIPTION
**Pull request by ehudon**
### Context

There is no JIRA associated this. This PR aims at adding `/api` in the path for the API documentation. 
image

#### Changes made

- Add `/api` to the endpoint used in the documentation.

### CODE REVIEW

File: source/includes/gcp/_networks.md [hunk 1]
```diff
@@ -85,14 +85,14 @@ curl -H "MC-Api-Key: your_api_key" \
...
-        "https://www.googleapis.com/compute/v1/projects/test-area-oox/regions/us-west1/subnetworks/default",
-        "https://www.googleapis.com/compute/v1/projects/test-area-oox/regions/northamerica-northeast1/subnetworks/default",
-        "https://www.googleapis.com/compute/v1/projects/test-area-oox/regions/us-east4/subnetworks/default"
+        "https://www.googleapis.com/compute/api/v1/projects/test-area-oox/regions/us-east1/subnetworks/default",
+        "https://www.googleapis.com/compute/api/v1/projects/test-area-oox/regions/us-central1/subnetworks/default",
+        "https://www.googleapis.com/compute/api/v1/projects/test-area-oox/regions/us-west2/subnetworks/default",
+        "https://www.googleapis.com/compute/api/v1/projects/test-area-oox/regions/us-west1/subnetworks/default",
+        "https://www.googleapis.com/compute/api/v1/projects/test-area-oox/regions/northamerica-northeast1/subnetworks/default",
+        "https://www.googleapis.com/compute/api/v1/projects/test-area-oox/regions/us-east4/subnetworks/default"
       ],
```
Comment (pdube) [inline source/includes/gcp/_networks.md]: All of these google api changes should be reverted. Try doing a replacement from `cloudmc_endpoint/v1` -> `cloudmc_endpoint/api/v1` instead

File: source/includes/gcp/_images.md [hunk 2]
```diff
@@ -112,7 +112,7 @@ Attributes | &nbsp;
 
 ```shell
 curl -H "MC-Api-Key: your_api_key" \
-    "https://api.your.cloudmc/v1/services/gcp/test-area/images/4658005417542122143"
+    "https://api.your.cloudmc/api/v1/services/gcp/test-area/images/4658005417542122143"
```
Comment (pdube) [inline source/includes/gcp/_images.md]: Should `api.your.cloudmc` be `cloudmc_endpoint` instead for consistency?

File: source/includes/gcp/_regions.md [hunk 3]
```diff
@@ -70,7 +70,7 @@ Attributes | &nbsp;
 
 ```shell
 curl -H "MC-Api-Key: your_api_key" \
-    "https://api.your.cloudmc/v1/services/gcp/test-area/regions/1330"
+    "https://api.your.cloudmc/api/v1/services/gcp/test-area/regions/1330"
```
Comment (pdube) [inline source/includes/gcp/_regions.md]: There's another one here

File: source/includes/kubernetes/_k8_persistentvolumeclaims.md [hunk 4]
```diff
@@ -116,7 +116,7 @@ curl -X GET \
       "name": "cmc-staging-mysql",
       "namespace": "cmc-stg",
       "resourceVersion": "111495311",
-      "selfLink": "/api/v1/namespaces/cmc-stg/persistentvolumeclaims/cmc-staging-mysql",
+      "selfLink": "/api/api/v1/namespaces/cmc-stg/persistentvolumeclaims/cmc-staging-mysql",
```
Comment (pdube) [inline source/includes/kubernetes/_k8_persistentvolumeclaims.md]: Double `/api` here

File: source/includes/kubernetes/_k8_persistentvolumeclaims.md [hunk 5]
```diff
@@ -285,7 +285,7 @@ Required Attributes                 | &nbsp;
 | `spec.volumemode` <br/>_string_        | What type of volume is required by the claim. Value of Filesystem is implied when not included in claim spec.|
 | `spec.dataSource` <br/>_string_        | This field can be used to specify either: an existing VolumeSnapshot object, an existing PVC or an existing custom resource/object that implements data population.|
 
-Note that the list is not complete, since it is refering to the [kubernetes api details](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#persistentvolumeclaim-v1-core).
+Note that the list is not complete, since it is refering to the [kubernetes api details](https://kubernetes.io/docs/reference/generated/kubernetes-api/api/v1.19/#persistentvolumeclaim-v1-core).
```
Comment (pdube) [inline source/includes/kubernetes/_k8_persistentvolumeclaims.md]: This should be reverted

**Review by pdube — CHANGES_REQUESTED** [review]
The search and replace done was a bit too broad. Pre-pending the search and replace with `cloudmc_endpoint/` should fix all of these

File: source/includes/gcp/_networks.md (hunk 1 above)
Comment (ehudon) [reply source/includes/gcp/_networks.md]: 🤦 Oops.

Comment (ehudon) [reply source/includes/gcp/_networks.md]: Fixed in commit b72186371f3f9263f21b901b1608cb3131183861

File: source/includes/gcp/_images.md (hunk 2 above)
Comment (ehudon) [reply source/includes/gcp/_images.md]: Fixed in commit b72186371f3f9263f21b901b1608cb3131183861

File: source/includes/kubernetes/_k8_persistentvolumeclaims.md (hunk 4 above)
Comment (ehudon) [reply source/includes/kubernetes/_k8_persistentvolumeclaims.md]: Fixed in commit b72186371f3f9263f21b901b1608cb3131183861

File: source/includes/kubernetes/_k8_persistentvolumeclaims.md (hunk 5 above)
Comment (ehudon) [reply source/includes/kubernetes/_k8_persistentvolumeclaims.md]: Fixed in commit b72186371f3f9263f21b901b1608cb3131183861

**Comment by ehudon** [comment]
@pdube I have made the required fixes. Sorry about that.

**Review by franzpgarcia — APPROVED** [review]

**Review by pdube — APPROVED** [review]
Small typo otherwise LGTM 🚀

File: source/includes/gcp/_images.md [hunk 6]
```diff
@@ -8,7 +8,7 @@ Images are virtual machine images that have a virtual disk which contains a boot
 
 ```shell
 curl -H "MC-Api-Key: your_api_key" \
-    "https://api.your.cloudmc/v1/services/gcp/test-area/images"
+    "https://cloudmc_endpoint/api/v1services/gcp/test-area/images"
```
Comment (pdube) [inline source/includes/gcp/_images.md]: Small typo here: `v1/services`

Comment (ehudon) [reply source/includes/gcp/_images.md]: Thanks @pdube. I fixed this one.

**Pull request merged**